docs/source/getting-started/index.rst

```diff
@@ -7,105 +7,109 @@ Getting Started with Daml
+The command has started successfully when you see the ``INFO  com.daml.http.Main$ - Started server: ServerBinding(/127.0.0.1:7575)`` message in the terminal. The command does a few things:
 
-    1. Compiles the Daml code to a DAR (Daml Archive) file.
-    2. Generates a JavaScript library in ``ui/daml.js`` to connect the UI with your Daml code.
-    3. Starts an instance of the :doc:`Sandbox </tools/sandbox>`, an in-memory ledger useful for development, loaded with our DAR.
-    4. Starts a server for the :doc:`HTTP JSON API </json-api/index>`, a simple way to run commands against a Daml ledger (in this case the running Sandbox).
+    1. Compiles the Daml code to a DAR (Daml Archive) file
+    2. Generates a JavaScript library in ``ui/daml.js`` to connect the UI with your Daml code
+    3. Starts an instance of the :doc:`Sandbox </tools/sandbox>`, an in-memory ledger useful for development, loaded with our DAR
+    4. Starts a server for the :doc:`HTTP JSON API </json-api/index>`, a simple way to run commands against a Daml ledger (in this case the running Sandbox)
 
 We'll leave these processes running to serve requests from our UI.
 
-In a second terminal, navigate to the ``create-daml-app/ui`` folder and use ``npm`` to install the project dependencies::
+5. In the second terminal, navigate to the ``create-daml-app/ui`` folder and use ``npm`` to install the project dependencies::
 
     cd create-daml-app/ui
     npm install
 
-This step may take a couple of moments (it's worth it!).
+This step may take a couple of moments.
 You should see ``success Saved lockfile.`` in the output if everything worked as expected.
 
-Now you can start the UI with::
+6. Start the UI with::
 
     npm start
 
 This starts the web UI connected to the running Sandbox and JSON API server.
 The command should automatically open a window in your default browser at http://localhost:3000.
+
 Once the web UI has been compiled and started, you should see ``Compiled successfully!`` in your terminal.
-If it doesn't, just open that link in a web browser.
-(Depending on your firewall settings, you may be asked whether to allow the app to receive network connections. It is safe to accept.)
-You should now see the login page for the social network. For simplicity of this app, there is no password or sign-up required.
-First enter your name and click *Log in*.
+If you don't, open http://localhost:3000 in a web browser.
+Depending on your firewall settings, you may be asked whether to allow the app to receive network connections. It is safe to accept.
+
+You should now see the login page for the social network. For simplicity, in this app there is no password or sign-up required.
+
+1. Enter a user name. Valid user names are bob, alice, or charlie (note that these are all lower-case, although they are displayed in the social network AI with the usual capitalization).
```

Comment:
```suggestion
1. Enter a user name. Valid user names are bob, alice, or charlie (note that these are all lower-case, although they are displayed in the social network AI by their alias instead of their user id with the usual capitalization).
```